app/locations/controllers.js

```diff
@@ -34,7 +34,7 @@ async function locations(req, res, next) {
       })
     )
 
-    userLocations = supplierLocations.flat()
+    userLocations = [...new Set(supplierLocations.flat())]
```

Comment:
https://caniuse.com/mdn-javascript_builtins_set

Nice, it's covered by all the browsers I think we're supposed to support.

However, I just noticed this - https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/.browserslistrc#L5

Our browser list config says we support last 2 versions of all browsers, and all versions of IE < 11 - which seems odd.

Reply:
yeah, we do, there's some people using edge in IE7 mode or something scary like that. However, this is on the server side, so I don't think the browser cares. It is, apparently, the most vanilla way to do it, so we should safe either way.